resolwe_bio/tests/processes/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+**************
+Procesor Tests
+**************
+
+ .. automodule:: server.tests.processor.base
```

Comment:
Am I right that this is a left-over from our previous internal version? If so, please, just remove these 2 lines.